tests/acceptance/features/component/signal_stability.feature

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+
+Feature: Signal Stability for fact notifications.
+  As a scalability manager user
+  I want to have a temporal window
+  in order to reset the grouping when the period between facts is higher than the configured one.
+
+
+  @happy_path @skip @bug @CLAUDIA-5528
+  Scenario: Facts grouping by window size is reset when the time between facts is higher than the configured one (10 seconds by default) (1/3).
+    Given the configured tenant-id is registered in CLOTO component
+    And   RabbitMQ consumer is looking into the configured message bus
+    And   the context notification has default context elements
+    And   window size is set to "3"
```

Comment:
You forget to put explicitly the default time size of the windows in the Ands of the 3 scenarios.